<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formMerge3GrayIntoARgb</class>
 <widget class="QDialog" name="formMerge3GrayIntoARgb">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create RGB from 3 Gray Images</string>
  </property>
  <widget class="QLineEdit" name="txtRed">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>20</y>
     <width>451</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbRed">
   <property name="geometry">
    <rect>
     <x>532</x>
     <y>20</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelRed">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>24</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Red</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbGreen">
   <property name="geometry">
    <rect>
     <x>532</x>
     <y>66</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtGreen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>66</y>
     <width>451</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelGreen">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Green</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtBlue">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>111</y>
     <width>451</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbBlue">
   <property name="geometry">
    <rect>
     <x>532</x>
     <y>111</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBlue">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>115</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Blue</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSave">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>160</y>
     <width>181</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
